File: Level_D/d041.cs
using System;

public class Pz{
    public static void Main(){
        string[] arr = Console.ReadLine().Trim().Split(' ');
        int n = int.Parse(arr[0]);
        int d = int.Parse(arr[1]);
        int e = int.Parse(arr[2]);
        Console.WriteLine((d * e >= n) ? "OK" : "NG");
    }
}

File: Level_D/Python2/d041.py
# coding: utf-8

if __name__=='__main__':
    arr = raw_input().strip().split(' ')
    n = int(arr[0])
    d = int(arr[1])
    e = int(arr[2])
    if n <= d * e:
        print('OK')
    else:
        print('NG')
